### PR TITLE
Native WorksForYou VC

### PR DESCRIPTION
**Pull request by mennenia**
In order for me and @sarahscott to PR our changes as we go forward, I propose merging a switch between the working martsy view and our WIP native view. 

I've created a dev options (shake phone / enter ` in the simulator) to get the native works for you view (basically copied what was done for the native auctions view). So you can choose to continue with the webview or see what we're up to in the native view. 

This view is still empty at the moment and only has a title. I've started on the layout and Sarah is working on the network part, but that's not part of this PR.

### CODE REVIEW

File: Artsy/View_Controllers/Notifications/ARWorksForYouViewController.m [hunk 1]
```diff
@@ -0,0 +1,45 @@
+//
+//  ARNotificationsViewController.m
+//  Artsy
+//
+//  Created by Maxim Cramer on 16/02/2016.
+//  Copyright © 2016 Artsy. All rights reserved.
+//
```
Comment (orta) [inline Artsy/View_Controllers/Notifications/ARWorksForYouViewController.m]: remove the comments please

File: Artsy/View_Controllers/Notifications/ARWorksForYouViewController.m [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    ARSerifLabel *titleLabel = [[ARSerifLabel alloc] initWithFrame:CGRectZero];
+    // TODO: Localise / put strings elsewhere
+    titleLabel.text = @"Works by artists you follow";
+    titleLabel.textColor = [UIColor blackColor];
+
+    [stack addSubview:titleLabel withTopMargin:@"40"];
+}
+
+
+@end
```
Comment (orta) [inline Artsy/View_Controllers/Notifications/ARWorksForYouViewController.m]: you have swift available to you too

File: Artsy/View_Controllers/Notifications/ARWorksForYouViewController.m (hunk 1 above)
Comment (mennenia) [inline Artsy/View_Controllers/Notifications/ARWorksForYouViewController.m]: :+1: So'z removed them in a different branch but forgot to do so here

File: Artsy/View_Controllers/Notifications/ARWorksForYouViewController.m [hunk 3]
```diff
@@ -0,0 +1,37 @@
+#import "ARWorksForYouViewController.h"
+#import "ORStackView.h"
```
Comment (orta) [inline Artsy/View_Controllers/Notifications/ARWorksForYouViewController.m]: This is a Pod, should use `</>` - you can then ditch the `#import <FLKAutoLayout/UIView+FLKAutoLayout.h>` I think too

**Comment by ArtsyMobile** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%">1 Warning</th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:warning:</td>
      <td><p>Needs testing on a Phone if change is non-trivial</p>
</td>
    </tr>
    
  </tbody>
</table>
  

  


<p align="right" data-meta="generated_by_danger" data-base-commit="" data-head-commit="" >
  Generated by :no_entry_sign: <a href="https://github.com/KrauseFx/danger/">danger</a>
</p>

**Comment by alloy** [comment]
After removing the import, can you do a rebase?

**Comment by mennenia** [comment]
all done ^_^

**Comment by alloy** [comment]
Something must have gone wrong during that rebase, as the diff seems way too large. Maybe it’s a good idea to make a new branch off of the current master branch and then cherry-pick the commits you really made onto that on.

**Pull request closed without merging**